doc/lua_api.txt

```diff
@@ -3512,6 +3512,12 @@ Helper functions
 * `minetest.pointed_thing_to_face_pos(placer, pointed_thing)`: returns a
   position.
     * returns the exact position on the surface of a pointed node
+* `minetest.get_tool_wear_after_use(uses [, initial_wear])`
+    * Simulates a tool being used once and returns the added wear,
+    * such that, if only this function is used to calculate wear,
```

Comment:
Broken bullet points. Should only have a `*` on new bullet points, other lines should omit the * but line up the text